src/components/atoms/Heading/Heading.jsx

```diff
@@ -1,6 +1,6 @@
 import "./Heading.scss";
 
-const Heading =({ data })=>{
+const Heading = ({ data }) =>{
```

Comment:
Why do you have this kind of expression as parameter to component? I would suggest to pass `props` instead of `{data}` and then access data as `props.data`. The expression you used is beneficial when you what to get rid of `props.something` pattern you write `{something1, something2, something3}` and access it directly in your component

Reply:
Changed